Doc/library/importlib.rst

```diff
@@ -1429,6 +1429,11 @@ an :term:`importer`.
    ``/foo/bar/__pycache__/baz.cpython-32.opt-2.pyc``. The string representation
    of *optimization* can only be alphanumeric, else :exc:`ValueError` is raised.
 
+   The *noopt* parameter is used to specify if compiler optimization are
+   disabled. If it is true, *optimization* is ignored and ``.noopt`` suffix is
```

Comment:
"*optimization* is ignored" is good but please also add: "disable compiler optimizations"

Reply:
> My brain always fail to understand a double negation: `noopt=False`. What about:
> 
> * `use_optimization=True`
> * `with_optimization=True`
> * `enable_optimization=True`
> * `disable_optimization=False` (still double negation)
> * `optimization=True` ... the annoying part with this one is that compile() already has `optimize` parameter and there is already `sys.flags.optimize`, whereas cache_from_source() already has `optimization` :-(
> 
> To me, `with_optimization=True` makes sense (optimize), as well as `with_optimization=False` (don't optimize).
> 
> `with` fits well with an action like "compile". But it doesn't fit well with the global sys.flags. Maybe `use_optimization=True` is better.

I was considering doing this but the problem is that all options are too similar to `optimization=...` and that raises more confusion due to 'what do I have an option to set the optimization and another to deactivate the option I just used?'. Given that using `disable_optimizations=False` will likely never be used (the default is `None`) I would prefer to use `disable_optimizations`.

Another option is to deprecate `optimization` (still keeping it), rename it to `optimization_level` and then using `enable_optimization`. What do you think?